ADR/####-slsa-source-provenance-verification.md

```diff
@@ -0,0 +1,130 @@
+# ####. SLSA Source Provenance Verification
+
+Date: 2025-10-09
+
+## Status
+
+Proposed
+
+## Context
+
+Konflux's trust model is organized entirely around SLSA build provenance. Build provenance attestations, generated by Tekton Chains and stored alongside container images in OCI registries, form the foundation of our verification and release policies in Enterprise Contract/Conforma. This approach follows the SLSA principle of storing attestations close to the artifacts they describe.
+
+The SLSA specification includes a parallel track - the Source Track - which provides increasing levels of trust in source code by verifying source control system practices. The source track addresses threats like unauthorized source modifications, bypassed code reviews, and compromised version control systems. Just as build attestations are stored alongside the artifacts they describe (container images in OCI registries), the SLSA community proposes storing source verification summary attestations (VSAs) as git notes in source repositories, keeping attestations close to the source commits they verify.
+
+However, Konflux's build-provenance-centered architecture prompts a question: **How should we verify and incorporate source provenance into our existing trust model?**
+
+Some different approaches are possible:
+
+1. **Direct reference**: Include source provenance attestations directly in build provenance
+2. **Parallel verification**: Verify source provenance independently alongside build provenance
+3. **Chained attestations**: Create build provenance attestations about source provenance attestations
+
+The challenge is that the source track's storage model (git notes in repositories) differs significantly from our build artifact storage model (OCI registries), making it non-obvious how to integrate source provenance verification into our existing Conforma policies and attestation chains.
+
+### References
+
+- [SLSA Source Track Requirements](https://slsa.dev/spec/draft/source-requirements) - Source provenance specification
```

Comment:
The draft spec is more accurate even if it is moving. The RC1 spec is out of date. We should update this to either the RC2 reference (once created) or just wait until we have created the official SLSA v1.2 release.